### DISCUSSION TITLE
Laser not working

### DISCUSSION THREAD

**Discussion by ramyasreeadari** [arkypita/LaserGRBL#908] [Q&A]
I have bought CNC 3018 pro machine . I have mounted the laser and gave all the connections accordingly, and installed all the software and drivers in my laptop .now I started opening the software checked all the setting and gave the test size than the laser spot shows the test size on the workpiece,later on by clicking on the start button for laser engraving the X And Y axis adjust themselves to start the work ,the program is running but the laser is not burning as soon as I click the start button

**Comment by taz101** [comment]
See if you can increase power to 1000. 255 did not work for my laser.

**Comment by chasehensley** [comment]
> See if you can increase power to 1000. 255 did not work for my laser.

Hey different person but! I am also having this issue and mine works when I set it to 1000 but not any lower why is this?

**Comment by StuartB4** [comment]
@ramyasreeadari.
I think you have to enable Laser Mode, $32 on that machine to use the laser, then disable it to use the spindle motor.

**Comment by mjscholp** [comment]
> @ramyasreeadari.
> I think you have to enable Laser Mode, $32 on that machine to use the laser, then disable it to use the spindle motor.

how do you do that? Right now it is set to 1, i thought that was enabled?I just purchased this and am new to all of this. I am unsure if anything is enabled or disabled.

**Comment by StuartB4** [comment]
Yes, set to 1 is enabled. 
Also $30 should be 255.

**Comment by mjscholp** [comment]
> Yes, set to 1 is enabled.
> Also $30 should be 255.

Yep, that's what I have and still nothing. Thank you

**Comment by StuartB4** [comment]
@ramyasreeadari 
@mjscholp
Can you post some clear pics of your controller board and laser showing the connections.

**Comment by mjscholp** [comment]
![20200410_113443 1](https://user-images.githubusercontent.com/63422415/79007667-429dda80-7b21-11ea-90fe-c3b883e28c72.jpg)
![20200410_113313 1](https://user-images.githubusercontent.com/63422415/79007565-05394d00-7b21-11ea-90f1-14be52cb77ee.jpg)
![20200410_113345 1](https://user-images.githubusercontent.com/63422415/79007609-1e41fe00-7b21-11ea-8c10-154e2e6e4168.jpg)
![20200410_113515 1](https://user-images.githubusercontent.com/63422415/79007636-3285fb00-7b21-11ea-9006-b6419f2d97e8.jpg)

**Comment by StuartB4** [comment]
@mjscholp 
Sorry I cant read any of the writing on the underside of the board, too blurred. Also, theres a white wire coming from the laser which is usually the PWM wire, but I can't see where it goes to on the board, does it change colour somewhere?

**Comment by mjscholp** [comment]
![20200410_141353 1](https://user-images.githubusercontent.com/63422415/79016988-e72a1780-7b35-11ea-99ff-cf28043424a7.jpg)
![20200410_141420 1](https://user-images.githubusercontent.com/63422415/79017020-f5783380-7b35-11ea-80ba-8a95eb5a2193.jpg)
![20200410_141436 1](https://user-images.githubusercontent.com/63422415/79017051-032db900-7b36-11ea-9177-879ba880e7b8.jpg)
![20200410_141443 1](https://user-images.githubusercontent.com/63422415/79017059-0b85f400-7b36-11ea-869a-f275f0c8d8a5.jpg)

I dont think they change colors. They called out the pin# for connection, not saying that something couldn't be wrong but it is installed and connected per the instructions. Ill keep looking and see what i can find. thank you for taking the time.

**Comment by mjscholp** [comment]
> @mjscholp
> Sorry I cant read any of the writing on the underside of the board, too blurred. Also, theres a white wire coming from the laser which is usually the PWM wire, but I can't see where it goes to on the board, does it change colour somewhere?

I found the issue, of course user error. there was 1 connector not connected! thank you for taking the time

**Comment by StuartB4** [comment]
Ok mjscholp. 
Glad you got it sorted. :+1:
